Remotion/Web/Core/UI/Controls/WebButton.cs

```diff
@@ -215,8 +229,13 @@ protected override void AddAttributesToRender (HtmlTextWriter writer)
 
       ControlStyle.CssClass = computedCssClass;
 
+      var currentText = Text;
+      base.Text = Text.GetValue();
+
```

Comment:
```suggestion
      var currentText = Text;
      if (Text.IsEmpty && _icon != null && _icon.HasRenderingInformation)
        base.Text = _icon.AltText
      else
        base.Text = Text.GetValue();
```

Please check the IF-semantics, it's just a hip-shot right now.

Reply:
if `!Text.IsEmpty`, `Text` should be set alright.